packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/UpgradeRolesForm/UpgradeRolesForm.tsx

```diff
@@ -31,6 +37,33 @@ export const UpgradeRolesForm = ({
     { id: 2, checked: false },
   ]);
 
+  let upgradedUser;
+  let newRoleToBeUpgraded;
+  const membersRef = useRef();
+
+  const { useUpgradeRolesMutation } = updateRoles;
```

Comment:
see my other comment and see other files in state/api/ how the hooks are exported and used in compoennts